src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
+    pub bytes_per_line: usize,
+    /// Number of bytes per group
+    pub bytes_per_group: usize,
+    /// Separation between groups
+    pub chars_per_spacing: usize,
+    /// Separator between the address and data
```

Comment:
Controls the separator between the address label and the hex output of the data.

Default is ` | `